OKTA-804418 Deprecate Apps API: update /apps API links to Redocly (#5092)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md

Okta recommends that you upgrade SAML 2.0 app integrations that use SHA1 certificates to use SHA256 certificates instead. SHA256 is a more secure cryptographic hash function that superseded SHA1 in 2002. If your ISV doesn't accept certificates with an SHA256 signature, you can continue to use the previous SHA1 certificate. This guide also covers how to revert your app integration back to its original SHA1 certificate if there’s an issue.

> You can upgrade and revert certificates in the Admin Console, and also programmatically using the [Applications API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/). This guide covers both options.

Reviewer: ```suggestion
You can upgrade and revert certificates in the Admin Console, and also programmatically using the [Apps API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/). This guide covers both options.
```

> ### Generate a new app key credential

Reviewer: ```suggestion
### Generate a new application key credential
```

> [Generate a new X.509 certificate for an application key credential](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationSSOCredentialKey/#tag/ApplicationSSOCredentialKey/operation/generateApplicationKey), and make a note of the key ID `kid` value that is returned.

Reviewer: ```suggestion
[Generate a new X.509 certificate for an application key credential](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationSSOCredentialKey/#tag/ApplicationSSOCredentialKey/operation/generateApplicationKey) and make a note of the key ID `kid` value that's returned.
```

### Update the key credential for the app with the new signing key ID

After you create a new key credential for the app, you must update the app to use it.

> > **Note:** To update an application with the newly generated key credential, use the [Replace an Application](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication) request with the new [credentials.signing.kid](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication!path=4/credentials/signing/kid&t=request) value in the request body. You can provide just the [Signing Credential object](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication!path=4/credentials/signing&t=request) instead of the entire [Application Credential object](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication!path=4/credentials&t=request).

Reviewer: ```suggestion
> **Note:** To update an app with the newly generated key credential, use the [Replace an Application](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication) request with the new [credentials.signing.kid](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication!path=4/credentials/signing/kid&t=request) value in the request body. You can provide just the [signing credential object](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication!path=4/credentials/signing&t=request) instead of the entire [application credential object](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/replaceApplication!path=4/credentials&t=request).
```
